src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -0,0 +1,36 @@
+---
+group: frontend-developer-guide
+title: Create a new layout
+functional_areas:
+  - Frontend
+---
+
+### Create a new page layout in custom theme
+
+create a custom page-layout xml file in the below directory.
```

Comment:
We need to provide a little bit more information on why we need to create a new layout. For example, "... if a new page is going to be designed in 4-columns layout, you may create the new layout in the following way..."

Reply:
Also, I would recommend providing a better example than 4-column. Like `3-columns-double-footer`. The last one is more reasonable as a real-world example.

Reply:
@rogyar I have changed the example to 3-columns-double-footer as per your suggestion. Please review it.